cmd/vulcan-wpscan/wpscan.go

```diff
@@ -20,6 +21,10 @@ import (
 
 const (
 	NotAWordPressMessage = "The remote website is up, but does not seem to be running WordPress."
+	TargetResponding403  = "The target is responding with a 403"
+	TargetRedirectsTo    = "The URL supplied redirects to"
+
+	WPScanAPIStatusEndpoint = "https://wpscan.com/api/v3/status"
```

Comment:
exported constants should have a comment

Reply:
You are completely right. I've renamed the const vars to be "more goish".